product/reports/500_Events - Operations/150_AutoSDE_events.yaml

```diff
@@ -0,0 +1,72 @@
+---
```

Comment:
@OrGur1987 reports are already pluggable so you can move this to your plugin under a `content/reports/` directory

Reply:
ok, I'll open a new PR in our repo, removed the report from the main repo
the widgets stay here in the main repo though, right?

Reply:
new PR for the report in AutoSDE repo:
https://github.com/ManageIQ/manageiq-providers-autosde/pull/175

Reply:
Hey @OrGur1987 @nasark has completed making the widgets pluggable as well so you should be able to move all of this to your provider PR.  Place them in `content/dashboard/widgets`

Reply:
nice!
in that case I'm closing this PR,
all files are in AutoSDE repo in this PR:
https://github.com/ManageIQ/manageiq-providers-autosde/pull/175